### PR TITLE
Increase time a facehugger spends idle after trying to make friends with your face and failing to do so

### PR DESCRIPTION
**Pull request by gbasood**
#14598 made me look over the facehugger code again and I realized something
1.5 seconds between pounces from the same facehugger? That's REALLY DUMB
Let's change that.
🆑 
 - tweak: Increase time a facehugger spends idle after trying to make friends with your face and failing to do so (changed from 1.5 to 5 seconds of idle time)

### CODE REVIEW

**Review by PJB3005 — APPROVED** [review]

**Comment by Blithering** [comment]
I thought they usually kill themselves in the attempt?

**Comment by gbasood** [comment]
75% of the time.
If they're Lamarr or otherwise sterile it's actually just gonna get up and grab your face again every 1.5 secs.

When it comes to ayys throwing 4 facehuggers at you though, you're very likely to get double hugged at some point.

File: code/modules/mob/living/carbon/alien/special/facehugger.dm [hunk 1]
```diff
@@ -249,15 +249,15 @@ var/const/MAX_ACTIVE_TIME = 400
 			if(prob(rng)) // Temporary balance change, all mouth-covering hats will be more effective
 				H.visible_message("<span class='danger'>\The [src] smashes against [H]'s \the [mouth_protection], and rips it off in the process!</span>")
 				H.drop_from_inventory(mouth_protection)
-				GoIdle(15)
+				GoIdle(50)
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: Make it a define.

**Review by ComicIronic — CHANGES_REQUESTED** [review]
Define it.

File: code/modules/mob/living/carbon/alien/special/facehugger.dm (hunk 1 above)
Comment (PJB3005) [reply code/modules/mob/living/carbon/alien/special/facehugger.dm]: comic'd

**Review by ComicIronic — APPROVED** [review]

**Pull request merged**